frontend/src/components/SelectedProjectPage.tsx

```diff
@@ -23,15 +24,16 @@ function SelectedProjectPage() {
     repIds: [],
     relIds: [],
   } as Project);
-  const [reports, setReports] = useState([]);
-  const [relationships, setRelationships] = useState([]);
+  const [reports, setReports] = useState<any[]>([]);
```

Comment:
Small nit: can you try using the `ReportProperties` and `RelationshipProperties` type rather than just `any`? Since we do have types defined for it. If it breaks anything though, just stick w this, no need to waste time lol.

Reply:
I keep getting errors when I do that so I shall fix that later if necessary